docs/dev/README.md

```diff
@@ -32,6 +32,10 @@ In order to run the Tekton Dashboard, please make sure the requirements in [the
 You will also need the following tools in order to build the Dashboard locally and deploy it:
 1. [`go`](https://golang.org/doc/install): The language the Tekton Dashboard backend is built in
 1. [`git`](https://help.github.com/articles/set-up-git/): For source control
+
+   **Note:** On Windows make sure to set `core.autocrlf` to `true` in your git config to ensure line endings are handled correctly
```

Comment:
```suggestion
   **Note:** On Windows make sure to set `core.autocrlf` to `input` or `false` in your git config to ensure line endings are handled correctly
```

Reply:
we don't want to use `false` as that would allow users commit changes with carriage returns to the repo

`input` does seem to make more sense here, but we still have the problem with the prettier config
I'd prefer to keep our existing prettier config, let me see if there's something we can do here

Reply:
Added `.gitattributes` to enforce use of LF, git will auto-convert any CRLF to LF on commit (same as `core.autocrlf=input`), and will NOT convert to CRLF on checkout.

We already have `.editorconfig` which is supported by most of the popular editors, e.g. VS Code, Sublime Text, Atom, IntelliJ IDEA, Eclipse, WebStorm, NetBeans, etc. which sets line endings to LF, and our prettier config will catch any CRLF that manage to sneak through despite all of this.